cordova-lib/src/cordova/create.js

```diff
@@ -191,8 +191,8 @@ function create(dir, optionalId, optionalName, cfg) {
 
                 return remoteLoad.gitClone(gitURL, branch).fail(
                     function(err) {
-                        events.emit('verbose', err);
-                        return Q.reject('Failed to retrieve '+ cfg.lib.www.url + ' using git.');
+                        events.emit('warn', err.message);
+                        return Q.reject(new CordovaError('Failed to retrieve '+ cfg.lib.www.url + ' using git.'));
```

Comment:
Should the `err.message` be appended to new CordovaError or the original `err` be passed as inner error?